GitForWindowsHelper/slash-commands.js

```diff
@@ -105,6 +105,35 @@ module.exports = async (context, req) => {
             return `I edited the comment: ${commentURL}`
         }
 
+        if (command == '/updpkgsums') {
+            if (owner !== 'git-for-windows'
+             || !req.body.issue.pull_request
```

Comment:
We should move the requirement to be issued from a PR to its own error mesage, here and in other slash commands at some point in the future.

\#leftoverbits